### ISSUE TITLE
Slack notification formatting

### ISSUE THREAD

**Issue by wcpettus** [project8/hornet#40]
The values aren't being formatted into the Slack messages, but are getting pasted after:

```
NOTICE > Scheduler summary:
    In the past %v,
     - Scheduled %d file(s)
     - Finished %d file(s) 1m0s 3 3
```

**Comment by guiguem** [comment]
this is a problem of `debugf` instead of `debug`, I guess

**Comment by wcpettus** [comment]
This is fixed at least in v2.4.15
